### DISCUSSION TITLE
How to power on Pi4 when plugging directly into mains (power USB direct into Pi4, not PiJuiceHat), without having to press SW1 button.

### DISCUSSION THREAD

**Discussion by GooGooMa** [PiSupply/PiJuice#948] [Q&A]
I built a Lego TV Set with a Raspberry Pie and Retropie running inside it, normally the Pi powers on automatically as soon as I plug it into the mains.

I am new to the world of Pi, and just bought a PiJuiceHAT, so that I can move the unit around and play it wherever. 

The unit is still being powered directly to the Pi4, as I only have a USB-C charger, not a Micro USB that the PiJuiceHAT takes.

I've noticed that now when I turn it on at the mains, it no longer auto boots up. I've had to manually press SW1 - which is not practical as it means taking apart the unit to access it.

Could anyone tell me if there is a simple solution (perhaps in the settings?) to revert to the original boot up behaviour as primary wake up - i.e. just powering it on from the mains?

Thanks

**Comment by ChristopherRush** [comment]
Have you installed the PiJuice software? Use function wake up on charge
with a value of 1%. Otherwise you could connect a switch to header J4,
which replicates SW1.

**Comment by GooGooMa** [comment]
I installed the software so I shall try this. Due to the fiddly nature of
the set up the most convenient way to power on is at the plug socket, so
that is the solution I’m looking for!
Thanks
